### DISCUSSION TITLE
RAVEN code Tutorial

### DISCUSSION THREAD

**Discussion by DelayedNeutrons** [idaholab/raven#2397] [Q&A]
1. Hello, I wanted to know if there's any sort of tutorial or recorded workshop of Raven or not.
2. Is RELAP5/BISON mandatory for using RAVEN?
3. Does RAVEN take Excel sheets as input data?

**Comment by wangcj05** [comment]
FYI, you can find raven workshop materials at: https://github.com/idaholab/raven/tree/devel/doc/workshop 

RELAP5 and BISON are not required by RAVEN.  RAVEN can only accept CSV files as input, and RAVEN generated Database (HDF5) as inputs.

**Comment by DelayedNeutrons** [comment]
Dear Dr. Wang,

Thank you so much for your response. I truly appreciate it, and I am overwhelmed. Unfortunately, I don't have access to RELAP5; I only have access to PCTRAN.

In the workshop materials, I can only see PowerPoint files and codes, but I haven't found any video files. Are there any plans for future virtual workshops? If so, please count me in!


--
Sincerely,
Rubayet Alam
BSc in Nuclear Science and Engineering (NSE)
Military Institute of Science and Technology (MIST)
Dhaka, Bangladesh
www.rubayetalam.com<http://www.rubayetalam.com/>
________________________________

**Reply by wangcj05** [reply]
If you can share your email with me, I will add you to our mailing list.

**Comment by DelayedNeutrons** [comment]
Sure, It's ***@***.***


--
Sincerely,
Rubayet Alam
BSc in Nuclear Science and Engineering (NSE)
Military Institute of Science and Technology (MIST)
Dhaka, Bangladesh
www.rubayetalam.com<http://www.rubayetalam.com/>
________________________________

**Comment by wangcj05** [comment]
@DelayedNeutrons , For some reason, it didn't show correctly, I can only see ***@***.***. You can email me directly: Congjian.wang@inl.gov.
